sql/moz-fx-data-glam-prod-fca7/glam_etl/glam_desktop_beta_aggregates/view.sql

```diff
@@ -1,17 +1,93 @@
 CREATE OR REPLACE VIEW
-  `moz-fx-data-glam-prod-fca7.glam_etl.glam_desktop_beta_aggregates`
-AS
-SELECT
-  *
-FROM
-  `moz-fx-data-glam-prod-fca7.glam_etl.glam_desktop_beta_aggregates_v1`
-WHERE
+  `moz-fx-data-glam-prod-fca7.glam_etl.glam_desktop_beta_aggregates` AS (
```

Comment:
Just found interesting that we don't use the `_derived` convention in this project, I guess no need to separate tables and views.

Reply:
True. Still I think it would be beneficial to use the convention. Unfortunately it's not in the priority queue at the moment.